.evergreen/run-tests.sh

```diff
@@ -138,6 +138,8 @@ if [ "$CLIENT_SIDE_ENCRYPTION" = "on" ]; then
    wait_for_kms_server 7999
    wait_for_kms_server 8000
    wait_for_kms_server 8001
+   wait_for_kms_server 5698
+   wait_for_kms_server 8002
```

Comment:
```suggestion
   wait_for_kms_server 8002
   wait_for_kms_server 5698
```
Minor: order not consistent with scripts and documentation.